index.html

```diff
@@ -20,26 +21,161 @@
+                    <p class="techs">bootstrap</p>
+                    <p class="techs">Ruby</p>
+                </div>
+                <input class="button-project" type="submit" value="See project">
+            </section> 
+            
+            <section class="projects-container">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="techs-container">                
+                    <p class="techs">html</p>
+                    <p class="techs">bootstrap</p>
+                    <p class="techs">Ruby</p>
+                </div>
+                <input class="button-project" type="submit" value="See project">
+            </section> 
+            
+            <section class="projects-container">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="techs-container">                
+                    <p class="techs">html</p>
+                    <p class="techs">bootstrap</p>
+                    <p class="techs">Ruby</p>
+                </div>
+                <input class="button-project" type="submit" value="See project">
+            </section> 
+            
+            <section class="projects-container">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="techs-container">                
+                    <p class="techs">html</p>
+                    <p class="techs">bootstrap</p>
+                    <p class="techs">Ruby</p>
+                </div>
+                <input class="button-project" type="submit" value="See project">
+            </section>
         </section>        
-        <ul class="social-media">
-            <li>
-                <a href="#"><img class="github" src="img/github.png" alt="github"></a>
-            </li>
-            <li>
-                <a href="#"><img class="vector" src="img/vector.png" alt="vector"></a>
-            </li>
-            <li>
-                <a href="#"><img class="twitter" src="img/twitter.png" alt="twitter"></a>
-            </li>
-            <li>
-                <a href="#"><img class="msocial" src="img/msocial.png" alt="github"></a>
-            </li>
-        </ul>
+        
+        <section class="about">
+            <h2>About me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <input type="submit" value="Get My Resume">
```

Comment:
- The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling if it's required.